dlt/common/destination/reference.py

```diff
@@ -493,19 +497,55 @@ def df(self, chunk_size: int = None) -> Optional[DataFrame]:
         """
         ...
 
-    def arrow(self, chunk_size: int = None) -> Optional[ArrowTable]: ...
+    # accessing data
+    def arrow(self, chunk_size: int = None) -> Optional[ArrowTable]:
+        """fetch arrow table of first 'chunk_size' items"""
+        ...
+
+    def ibis(self, chunk_size: int = None) -> Optional[IbisTable]: ...
```

Comment:
I'd add it on `ReadableDBAPIDataset` and return implementation of it in `_dataset` of pipeline